docs/settings.rst

```diff
@@ -31,9 +31,35 @@ click the *Promote to administrator* button.
 
 .. figure:: /_static/images/settings/user.png
 
+Replacements
+------------
+
 Administrators can also delete users,
 using the delete button next to the user name.
 
+If you want to delete a user account, for example,
+because the user is leaving your organization, unfinished tasks to this account may still exist.
+In this case, specify a replacement who will take over open task assignements.
+The substitute will then have exactly all access rights to complete these tasks,
+but not automatically inherits all rolls or rights of the deleted user.
```

Comment:
If rights means access rights, it's not correct. The substitute will inherit the access rights for open tasks and cases. If rights means group memberships then that's correct, they are not transferred.